renovate-config.js

```diff
@@ -55,6 +55,16 @@ function configureArtifactory() {
       ],
       name: "artifactory",
       config: {
+        secrets: {
+          ARTIFACTORY_AUTH_TOKEN: process.env.RENOVATE_ARTIFACTORY_PASSWORD,
+          ARTIFACTORY_USERNAME: process.env.RENOVATE_ARTIFACTORY_USERNAME,
+        },
+        customEnvVariables: {
+          ORG_GRADLE_PROJECT_artifactoryAuthToken:
```

Comment:
This conditions the name of the property a consumer repo needs to use. I could customize this via a GHA input